README.md

```diff
@@ -47,9 +47,29 @@ At first glance these appear similar, but there is a distinction: the default sa
 
 The idea is to make it clear to other developers who might work on your project what dependencies are required for it to work and what are only needed for the dev environment.
 
+### npm scripts
+
+You may have noticed that npm created a section called "scripts" in your `package.json`. These are handy shortcuts you can create to run dependencies you install. Think of them as per project command line aliases.
+
+npm installs executable packages into `node_modules/.bin`, which means you can run them via the command line like this (for example) `node_modules/.bin/cowsay hello`. npm scripts automatically include this directory, so you can instead add a test script like this:
+
+```json
+{
+  "scripts": {
+    "hello": "cowsay hello"
```

Comment:
Yeah changing the name of the script is a good idea.

I needed to use an npm package rather than something like `echo` to demonstrate that scripts alias the `node_modules/.bin` folder. I can add a link to Cowsay if that helps

Reply:
If cowsay is well known feel free to leave it as is. It might just be me... :)

Reply:
```suggestion
    "greeting": "cowsay hello"
```

Reply:
I think I'll leave it as a nice surprise when they run it the first time :)